server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionTestService.java

```diff
@@ -65,6 +67,8 @@ public ConnectionTestResult test(@NotNull @Valid TestConnectionReq req) {
         Long connectionId = req.getId();
         if (ConnectionAccountType.SYS_READ == req.getAccountType()) {
             req.setTenantName("sys");
+        } else if (StringUtils.equals("sys", req.getTenantName()) && req.getType() != ConnectType.OB_MYSQL) {
+            throw new BadRequestException("sys租户是一个 MySQL 兼容租户");
```

Comment:
please handle i18n for error message, 
you may refer ErrorCodes usage

Reply:
correct to ErrorCode ConnectionDatabaseTypeMismatched